### PR TITLE
SvelteKit release candidate migration: quiz page

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
- Moved relevant icons to /src/lib/components/svg/
- Moved relevant assets to /static/assets/quiz/
- Updated asset paths in src/lib/legacy/acm-paths.ts
- Migrated quiz…
  - Component to src/routes/quiz/quiz.svelte
  - Types to src/lib/public/quiz/questions/types.ts
  - Storage to src/lib/public/quiz/responses/storage.ts
  - Question data to src/lib/public/quiz/questions/data.ts
 - “More info” component to src/routes/quiz/more-info.svelte
 - Progress bar to src/routes/quiz/progress-bar.svelte

### CODE REVIEW

File: src/routes/nodebuds/+page.svelte [hunk 1]
```diff
@@ -79,7 +79,7 @@
     max-width: 600px;
   }
 
-  section h2 {
+  section h1 {
```
Comment (EthanThatOneKid) [inline src/routes/nodebuds/+page.svelte:82]: This change was likely intended to be inside #555.

**Review by AlanCortez1337 — APPROVED** [review]
Beautiful ✨looks like this quiz stuff is ready for migration!

File: src/routes/nodebuds/+page.svelte (hunk 1 above)
Comment (EthanThatOneKid) [reply src/routes/nodebuds/+page.svelte:82]: This is minor, so it's fine.

**Comment by EthanThatOneKid** [comment]
Replaced by mega-PR #577.

**Pull request closed without merging**